### ISSUE TITLE
Malformatted if EOL is CRLF

### ISSUE THREAD

**Issue by weberjn** [danny0838/git-store-meta#42]
on the [mys2](https://www.msys2.org/) environment the script gives 
```
weberjn@julio MSYS ~/tmp/playground
$ git-store-meta.pl -a
applying metadata from `/home/weberjn/tmp/playground/.git_store_meta' ...
error: `/home/weberjn/tmp/playground/.git_store_meta' is malformatted.

weberjn@julio MSYS ~/tmp/playground
$ cat .git_store_meta
# generated by  git-store-meta  2.3.4
<file>  <type>  <mtime>
a       f       2022-10-28T20:53:29Z
b       f       2022-10-28T20:57:26Z

weberjn@julio MSYS ~/tmp/playground
```
Looks like a CRLF/LF problem, it works after dos2unix  .git_store_meta
```
 git config --global core.autocrlf
true

hexdump -C .git_store_meta
00000000  23 20 67 65 6e 65 72 61  74 65 64 20 62 79 09 67  |# generated by.g|
00000010  69 74 2d 73 74 6f 72 65  2d 6d 65 74 61 09 32 2e  |it-store-meta.2.|
00000020  33 2e 34 0d 0a 3c 66 69  6c 65 3e 09 3c 74 79 70  |3.4..<file>.<typ|
00000030  65 3e 09 3c 6d 74 69 6d  65 3e 0d 0a 61 09 66 09  |e>.<mtime>..a.f.|
00000040  32 30 32 32 2d 31 30 2d  32 38 54 32 30 3a 35 33  |2022-10-28T20:53|
00000050  3a 32 39 5a 0d 0a 62 09  66 09 32 30 32 32 2d 31  |:29Z..b.f.2022-1|
00000060  30 2d 32 38 54 32 30 3a  35 37 3a 32 36 5a 0d 0a  |0-28T20:57:26Z..|
00000070
```

(guess mys2 might be out of scope, but this is how I use git under Windows)

**Comment by danny0838** [comment]
What have you done to generate the `.git_store_meta` file?

**Comment by weberjn** [comment]
created reposity in Linux, git-store-meta.pl --install + --store
pushed
cloned in Linux, all ok

cloned in msys2, -> problem

**Comment by danny0838** [comment]
This should be fixed in 2.3.6 (5f5b3ec0b3f57d863c5f35be42ce0c2ca605627e). Thanks for the report.

**Comment by weberjn** [comment]
2.3.6 works now under Msys2.
Thank you very much.
